src/Tools/dotnet-user-jwts/src/Commands/KeyCommand.cs

```diff
@@ -0,0 +1,75 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.CommandLineUtils;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Tools.Internal;
+
+namespace Microsoft.AspNetCore.Authentication.JwtBearer.Tools;
+
+internal sealed class KeyCommand
+{
+    public static void Register(ProjectCommandLineApplication app)
+    {
+        app.Command("key", cmd =>
+        {
+            cmd.Description = "Display or reset the signing key used to issue JWTs";
+
+            var resetOption = cmd.Option(
+                "--reset",
+                "Reset the signing key. This will invalidate all previously issued JWTs for this project.",
+                CommandOptionType.NoValue);
+
+            var forceOption = cmd.Option(
+                "--force",
+                "Don't prompt for confirmation before resetting the signing key.",
+                CommandOptionType.NoValue);
+
+            cmd.HelpOption("-h|--help");
+
+            cmd.OnExecute(() =>
+            {
+                return Execute(cmd.Reporter, cmd.ProjectOption.Value(), resetOption.HasValue(), forceOption.HasValue());
+            });
+        });
+    }
+
+    private static int Execute(IReporter reporter, string projectPath, bool reset, bool force)
+    {
+        if (!DevJwtCliHelpers.GetProjectAndSecretsId(projectPath, reporter, out var _, out var userSecretsId))
+        {
+            return 1;
+        }
+
+        if (reset == true)
+        {
+            if (!force)
+            {
+                reporter.Output("Are you sure you want to reset the JWT signing key? This will invalidate all JWTs previously issued for this project.\n [Y]es / [N]o");
```

Comment:
```suggestion
                reporter.Output("Are you sure you want to reset the JWT signing key? This will invalidate all JWTs previously issued for this project.{Environment.NewLine} [Y]es / [N]o");
```